spec/models/solidus_bolt/gateway_spec.rb

```diff
@@ -22,8 +33,24 @@
   end
 
   describe '#capture' do
-    it 'raises NotImplementedError' do
-      expect { described_class.new.capture(nil, nil, nil) }.to raise_error(NotImplementedError)
+    subject(:capture) { described_class.new.capture(amount, response_code, gateway_options) }
+
+    let(:response_code) { 'the_amazing_spiderman' }
+    let(:response) {
+      { 'reference' => response_code, 'status' => 'authorized' }
+    }
+
+    before do
+      allow(SolidusBolt::Transactions::CaptureService).to receive(:call).and_return(response)
+      allow(SolidusBolt::Transactions::DetailService).to receive(:call).and_return(response)
```

Comment:
This should be removed, right?

Reply:
Removed here
[aafaa3f](https://github.com/nebulab/solidus_bolt/commit/aafaa3fe8858a8e9321b5a22e19ee1926df4a168)